core/types/types.go

```diff
@@ -100,3 +102,9 @@ type MigrationMetadata struct {
 	GenesisConfig    []byte `json:"genesis_config"`    // GenesisConfig is the genesis file data
 	SnapshotMetadata []byte `json:"snapshot_metadata"` // SnapshotMetadata is the snapshot metadata
 }
+
+type SignedChallenge struct {
+	Challenge []byte          `json:"challenge"`
+	Signature *auth.Signature `json:"signature"`
+	Sender    []byte          `json:"sender"`
```

Comment:
As with `CallMessage.Sender`, this should be `HexBytes`.

BTW, it's a little unfortunate that a `CallMessage` has will repeat sender twice, once int the struct itself, and once in this field.  Thoughts about avoiding that?